ark/spLDA/processing.py

```diff
@@ -278,7 +279,8 @@ def compute_topic_eda(features, featurization, topics, silhouette=False, num_boo
     stats = dict(zip(stat_names, [{} for name in stat_names]))
 
     # iterative over topic number candidates
-    for k in topics:
+    pb_format = '{l_bar}{bar}| {n_fmt}/{total_fmt} [{elapsed}<{remaining}]'
+    for k in tqdm(topics, bar_format=pb_format):
```

Comment:
Did you run the notebook to make sure the display looks as expected?

Reply:
It does.